frontend/containers/discover-map/map-help/component.tsx

```diff
@@ -0,0 +1,207 @@
+import { FC, useState } from 'react';
+
+import { HelpCircle, Layers, MapPin, Search, ZoomIn, ZoomOut, X as XIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Modal from 'components/modal';
+import { theme } from 'tailwind.config';
+
+import clusterExemple from 'svgs/map/cluster-exemple.svg';
+
+import { MapHelpProps } from '.';
+
+export const MapHelp: FC<MapHelpProps> = () => {
+  const { formatMessage } = useIntl();
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="">
+      <Button
+        className="flex items-center justify-center w-8 h-8 px-1.5 py-1.5 text-gray-800 bg-white rounded shadow focus-visible:outline-green-dark"
+        theme="naked"
+        size="smallest"
+        icon={() => <HelpCircle className="mr-0" />}
+        onClick={() => setOpen(true)}
+        aria-label={formatMessage({ defaultMessage: 'Open map help', id: 'q/Z5g4' })}
+      />
+      <Modal
+        open={open}
+        onDismiss={() => setOpen(false)}
+        title={formatMessage({ defaultMessage: 'Map help', id: '5eVTzC' })}
+        theme="naked"
```

Comment:
Why have you used a “naked” theme? Is it because of overflow issues (you could add some horizontal padding)?

If you use the default theme, you already get the close button and don't need a complex layout inside the modal.